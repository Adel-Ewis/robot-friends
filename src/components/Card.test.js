import { shallow } from 'enzyme';
import Card from './Card';
import React from 'react';

describe('Testing Card Component', () => {
  it('expects to render card Component', () => {
    expect(shallow(<Card />)).toMatchSnapshot();
  });
});
