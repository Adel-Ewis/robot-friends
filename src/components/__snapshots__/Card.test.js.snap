// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Card Component expects to render card Component 1`] = `
<div
  className="bg-light-green dib br3 pa3 ma2  grow bw2 shadow-5 tc"
>
  <img
    alt="robots"
    src="https://robohash.org/undefined?size=200x200"
  />
  <div>
    <h2 />
    <p>
       
       
    </p>
  </div>
</div>
`;
